epochStart/bootstrap/process_test.go

```diff
@@ -90,8 +91,7 @@ func createMockEpochStartBootstrapArgs(
 ) ArgsEpochStartBootstrap {
 	generalCfg := testscommon.GetGeneralConfig()
 	nodesCoordinatorRegistryFactory, _ := nodesCoordinator.NewNodesCoordinatorRegistryFactory(
-		&testscommon.MarshalizerMock{},
-		&epochNotifier.EpochNotifierStub{},
+		&marshal.GogoProtoMarshalizer{},
```

Comment:
Yes, because `MarshalizerMock` uses json marshaller,.
`NodesCoordinatorRegistryFactory` uses 2 marshallers:
1. hard coded json marshaller 
2.  another marshaller which is expected to be != json (in our case, proto).

Decided to changed it to use this mock instead: `NodesCoordinatorRegistryFactoryMock`